deploy/charts/rook-ceph/values.yaml

```
@@ -77,9 +77,14 @@ containerSecurityContext:
 # -- If true, loop devices are allowed to be used for osds in test clusters
 allowLoopDevices: false
 
+ceph-csi-operator:
+  nameOverride: ceph-csi-operator
+  fullnameOverride: ceph-csi-operator
+
 # Settings for whether to disable the drivers or other daemons if they are not
 # needed
 csi:
+  rookUseCSIOperator: false
```

This capitalization seems more consistent with other csi settings
```suggestion
  rookUseCsiOperator: false
```

---

done